extensions/azurePublish/src/components/ChooseResourcesList.tsx

```diff
@@ -47,18 +47,18 @@ const ImageIconPlacholder = styled.div`
 
 const ItemText = styled(Text)`
   font-size: ${FluentTheme.fonts.mediumPlus.fontSize};
-  margin-left: 4px !important;
+  margin-left: 8px !important;
 `;
 
 const ItemTier = styled(Text)`
   font-size: ${FluentTheme.fonts.small.fontSize};
-  margin: 4px 0 0 22px;
+  margin: 4px 0 3px 24px;
```

Comment:
We should try to use multiples of 4px.
```suggestion
  margin: 4px 0 4px 24px;
```